src/map/entities/battleentity.h

```diff
@@ -291,6 +291,7 @@ enum class REACTION : uint8
     HIT     = 0x08, // Hit
     EVADE   = 0x09, // Evasion (MISS + HIT)
     ABILITY = 0x10, // Observed on JA and WS
+    DEFLECT = 0x0A, // Attack hit but no damage dealt because of status effect such as Granite Skin
```

Comment:
Wouldnt this (0x0A) come before  0x10?

Reply:
Yes, I don't know what I was thinking there. I completely removed this though since it is really just hit + parry.